api/session/auth_session.py

```diff
@@ -55,6 +55,8 @@ def clear_session():
 
         Returns: 302 redirect to signed-out page
         """
+        fund = None
+        round = None
```

Comment:
We should avoid using `round` as it shadows the builtin [round](https://docs.python.org/3/library/functions.html#round)

Reply:
great point! have changed this now 👍🏾